plugins.xml

```diff
@@ -940,6 +942,45 @@
 			<certification type="official" />
 			<description>Bugfix for password saving when using the settings form.</description>
 		</release>
+		<release date="2024-12-16" version="1.0.7.4" md5="7d6a2121e841dd75bdb5f372ecfa84e2">
+			<package>https://github.com/pkp/plagiarism/releases/download/v1_0_7-4/plagiarism-v1_0_7-4.tar.gz</package>
+			<compatibility application="ojs2">
+				<version>3.3.0.18</version>
```

Comment:
@touhidurabir, I forget the details here -- were core changes required that prevent this from being compatible with OJS 3.3.0-x prior to -18, and 3.4.0-x prior to -7?

Reply:
@asmecher I started working on this on `3.3.0-16` and `3.4.0-5` but finalize it by `3.3.0-18` and `3.4.0-7` . So I am not sure about other older version . May be Jason know a bit more about the compatibility of `3.3.0` as they run a test with a client but not sure about the `3.4.0` version of being used in any client testing . But I think they should be at compatible with `3.3.0-16` and `3.4.0-5` .

Reply:
OK, perhaps we can start with the builds you're comfortable with and expand to include more later -- that'll give us a better chance to gauge the community uptake of the upgrade process. But rather than listing versions one at a time, I'd suggest using the semver style matching. That would be `^3.3.0.16` and `^3.4.0.5`.

Reply:
updated .